service/templates/full_title.html

```diff
@@ -0,0 +1,223 @@
+<html lang="en">
+  <head>
+    <style type="text/css">
```

Comment:
Not necessarily. This css is specific to the generation of the PDF *only*.
I'd suggest keeping the PDF generation styles (and template) self contained makes more sense - to avoid confusion with the UI styles?